python/lsst/ts/m2gui/controltab/tab_alarm_warn.py

```diff
@@ -176,8 +176,7 @@ def _set_indicators_limit_switch(self, limit_switch_type):
 
         indicators = dict()
         for name in limit_switch_status.keys():
-            indicator = set_button(name, None, is_indicator=True)
-            indicator.adjustSize()
+            indicator = set_button(name, None, is_indicator=True, is_adjust_size=True)
```

Comment:
Do you really need that? I still don't see a reason for adjust_size call. There are wrong friends, making you assume you need something in fact when you don't need it. I prefer to drop is_adjust_size parameter unless there is a reason to use it - I don't see a difference in my App (running directly on Mac, but shall be same across platforms).

Reply:
I need it to recalculate the width of button for the adjustment to the minimum one. In my test, the Qt will not do this by itself if I am looking for some smaller width (minimum) than the default one (after it calculates the widths for all the buttons). Thanks!